### PR TITLE
ExternalActor

### PR DESCRIPTION
**Pull request by chronaeon**

### CODE REVIEW

**Comment by chronaeon** [comment]
@webthethird I applied your suggested changes to this one.

**Comment by chronaeon** [comment]
@webthethird once we have this done and submitted I hope you're ready to tackle the next two and get them commit-ready before Wednesday

**Comment by webthethird** [comment]
@chronaeon have you determined how we should go about crafting the second definition for each term?

**Comment by chronaeon** [comment]
@webthethird If we do it right the second definition should easily emerge from the first. Here is how:

1. Write the first definition for each term as detailed as possible for the smart, cool, and chic technical user.
2. For the 2nd definition, cherrypick the best words and phrases from the detailed definition, and optimize it to fit in a tooltip by writing one clear sentence for non-techies.

**Comment by jo-tud** [comment]
This is a trick to ignore the white spaces in the diff: just add `?w=1` to the end of the URL.
e.g. https://github.com/ConsenSys/EthOn/pull/114/files?w=1

**Comment by jo-tud** [comment]
This PR needs a fix, before it can be merged. It includes changes from other PRs, somehow.

**Comment by chronaeon** [comment]
I'm going to start editing only the XML file. Protege helped a lot at first to learn the data structures of EthOn.rdf, but now that I know them I can move on to the hardcore user's method :P. I must be doing something (or a plugin is) to make all those weird extra changes.

**Comment by chronaeon** [comment]
Alright, this one should be good to merge now! :+1:  :D

**Comment by chronaeon** [comment]
OK, how can I make it so that only my "Bam!_2" change gets committed? @webthethird

**Comment by chronaeon** [comment]
Changes completed,

File: EthOn.rdf [hunk 1]
```diff
@@ -1665,11 +1653,11 @@
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/AccountConcept"/>
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/State"/>
         <suggestedStringRepresentation xml:lang="en">AccountState</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">State of an Ethereum Account. It is comprised on four pieces of information: nonce, balance, storage root and code hash. The data is stored in a Merkle Patricia tree as a mapping between addresses and Account states. The Account State is part of the World State, as it resembles the state of exactly one Account.</rdfs:comment>
+        <rdfs:comment xml:lang="en">State of an Ethereum Account. It is comprised on four pieces of information: nonce, balance, storage root and code hash. The data is stored in a Merkle Patricia tree as a mapping between addresses and Account states. The Account State is part of the World State, as it resembles the state of exactly one Account. In EthOn the code hash is not part of the state but a property of a ContractAccount since it is static. </rdfs:comment>
```
Comment (hydrarulz) [inline EthOn.rdf]: Typo
> It is comprised ~~on~~ **of** four pieces of information

Consistency in writing **Account State** 
> addresses and Account **States**. The Account State

File: EthOn.rdf [hunk 2]
```diff
@@ -2220,7 +2149,8 @@
             </owl:Restriction>
         </rdfs:subClassOf>
         <suggestedStringRepresentation>Tx</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Transactions are Messages between two Accounts that may transfer Ether and may contain a payload. Transactions always originate from an External Account that is controlled by an External Actor by means of a private key.  The execution of a Transaction creates a &apos;Transaction Receipt&apos;.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Transactions are Messages between two Accounts that may transfer Ether and may contain a payload. Transactions always originate from an External Account that is controlled by an External Actor by means of a private key.  The execution of a Transaction creates a TransactionReceipt.</rdfs:comment>
```
Comment (hydrarulz) [inline EthOn.rdf]: Double space after `private key.` Doesn't seem to be displayed everywhere, it seems that the medium controls if double space is displayed or not (html ignores it sometimes) Should be fixed though.
> a private key.  The execution of

**Pull request closed without merging**